examples/android/lib/src/main/kotlin/examples/android/lib/MainActivity.kt

```diff
@@ -1,11 +1,11 @@
 package examples.android.lib
 
 import android.app.Activity
+import android.app.AlertDialog
 import android.os.Bundle
 import android.widget.Button
 import android.widget.LinearLayout
 import android.widget.LinearLayout.LayoutParams
-import androidx.appcompat.app.AlertDialog
```

Comment:
We leaned on appcompat to ensure it was included as part of the build.

Can you add something that requires it?